src/util.js

```diff
@@ -188,7 +190,9 @@ const IMPORTANT_RE = /^([^:]+:.*?)( !important)?;$/;
 
 // Given a single style rule string like "a: b;", adds !important to generate
 // "a: b !important;".
+/* eslint-disable no-unused-vars */ // forgive existing code
 export const importantify = (string) =>
     string.replace(
         IMPORTANT_RE,
         (_, base, important) => base + " !important;");
+/* eslint-enable no-unused-vars */
```

Comment:
`importantify` is being exported sort it shouldn't be triggering `no-unused-vars.  Can you try adding the `es6` environment to your .eslintrc.json to see if that clears this up?